guides/common/modules/proc_enabling-pull-based-transport-on-proxy.adoc

```diff
@@ -0,0 +1,44 @@
+[id="enabling-pull-based-transport-on-proxy_{context}"]
+= Enabling Pull-Based Transport on {SmartProxy}
+
+Use the following procedure to enable pull-based transport on {SmartProxy}.
+
+.Procedure
+
+. Enable the pull mode and enable the {SmartProxy} for host registration
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt
+----
+. To , run:
```

Comment:
Incomplete sentence. Also, I'm unsure this needs to be part of the procedure.

In the Satellite context templates and registration are actually on by default (unlike upstream) so there it is redundant.

I'd suggest to leave this out, but I may be missing something.

Reply:
Yes, you are missing that this PR is still a Draft. But thank you for your input.